sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -625,7 +638,11 @@ export class MessageSession extends LinkEntity<Receiver> {
               this.logPrefix,
               bMessage.messageId
             );
-            this._onError!(err);
+            this._onError!(err, {
```

Comment:
Same question as the one mentioned before for the non-sessions case.
onError should be called before the if-check?

Reply:
Removed, seems very odd.